node_modules/oae-tenants/lib/api.js

```diff
@@ -132,15 +132,23 @@ var init = module.exports.init = function(_serverConfig, callback) {
  * Get a list of all available tenants from cache. The global admin tenant will be excluded from the resulting tenant list
  *
  * @param  {Boolean}        [excludeDisabled]   Whether or not disabled tenants should be included. By default, all tenants will be returned
+ * @param  {Number}         [start]             The index to start paging at, defaults to 0
+ * @param  {Number}         [limit]             The maximum number of tenants to return, if not specified all tenants will be returned
  * @return {Object}                             An object keyed by tenant alias holding all the tenants
  */
-var getTenants = module.exports.getTenants = function(excludeDisabled) {
+var getTenants = module.exports.getTenants = function(excludeDisabled, start, limit) {
```

Comment:
We try to default the function parameters at the top of the function by using OaeUtil. So this could become:

```javascript
var getTenants = module.exports.getTenants = function(excludeDisabled, start, limit) {
    excludeDisabled = OaeUtil.castToBoolean(excludeDisabled);
    start = OaeUtil.getNumberParam(start, 0);
    // Note that we don't apply a default limit here as some methods need ALL tenants
    limit = OaeUtil.getNumberParam(limit);
```